test/metabase/upload_test.clj

```diff
@@ -1751,3 +1752,45 @@
                       (rows-for-table table)))
 
               (io/delete-file file))))))))
+
+(defn- upload-table-exists? [table]
```

Comment:
I guess this could reuse the helper method I made, but for now it's nice that this gives stronger confidence than a hacky method that string matches against per-driver exceptions.

We'll naturally be forced to generalize this when we extend uploads to non-JDBC drivers.